templates/istio.yaml

```diff
@@ -55,4 +55,12 @@ spec:
             topologyKey: node-role.kubernetes.io/control-plane
     istio_cni:
       enabled: true
+---
+apiVersion: v1
+kind: Secret
+data:
+  config: LSBqb2JfbmFtZTogJ2Vudm95LXN0YXRzJwogIG1ldHJpY3NfcGF0aDogL3N0YXRzL3Byb21ldGhldXMKICBrdWJlcm5ldGVzX3NkX2NvbmZpZ3M6CiAgLSByb2xlOiBwb2QKCiAgcmVsYWJlbF9jb25maWdzOgogIC0gc291cmNlX2xhYmVsczogW19fbWV0YV9rdWJlcm5ldGVzX3BvZF9jb250YWluZXJfcG9ydF9uYW1lXQogICAgYWN0aW9uOiBrZWVwCiAgICByZWdleDogJy4qLWVudm95LXByb20nCg==
```

Comment:
Hmm is this actual secret data?

Reply:
Nope. If you decode that you will see that it is just a very basic Prometheus scrape config. This is how it was set up for Linkerd in the past, from Bill's work. I'm guessing that the Prometheus chart requires it to be a Secret so that users can put secret values in there if they want to. But we don't need to do that.

Reply:
Ok, I was curious if it is not secret then can it be configmap. that's fine if the chart requires it to be a secret.

Reply:
I'll go look and see if the chart lets me give it a ConfigMap instead, because I agree that that would be better.

Reply:
Yeah the docs for the operator say that it has to be a Secret.